src/main/java/net/dv8tion/jda/core/requests/restaction/ChannelAction.java

```diff
@@ -127,6 +128,29 @@ public ChannelAction setParent(Category category)
         return this;
     }
 
+    /**
+     * Sets the position where the new Channel should be inserted into.
+     * This refers to the raw position value, not the computed (relative) position.
+     * <p>
+     * By default (or by providing this method with {@code null}),
+     * the position will automatically be computed based on the other channels.
+     *
+     * @param  position
+     *         The raw position value that should be used for the new Channel
+     *
+     * @throws IllegalArgumentException
+     *         If the provided position value is {@code <0}
+     *
+     * @return The current ChannelAction, for chaining convenience
+     */
+    @CheckReturnValue
+    public ChannelAction setPosition(Integer position)
```

Comment:
Just wondering, why `Integer` instead of `int`?

Reply:
cuz nullable

Reply:
The alternative would've been using `-1` for "not set" since that wouldn't be in the accepted range anyway... but i like `null` more... It also goes with how `slowmode`, `bitrate` and `userlimit` are handled already.

Reply:
👌